react/__tests__/Video.test.tsx

```diff
@@ -175,12 +175,9 @@ describe('Video logic', () => {
     expect(videoRef?.current?.currentTime).toBe(2)
 
     act(() => {
-      videoRef?.current?.dispatchEvent(
-        new window.Event('timeupdate', () => {
-          expect(trackBarRef?.current.value).toBe('2')
-          expect(current.currentTime).toBe(2)
-        })
-      )
+      videoRef?.current?.dispatchEvent(new window.Event('timeupdate'))
     })
+
+    expect(trackBarRef?.current.value).toBe('2')
```

Comment:
Remove it from act function call was intentional?

Reply:
It was. It wasn't being executed before because it was passed inside a function to an event that didn't expect the function.